pkg/server/sotw/v3/server.go

```diff
@@ -73,26 +73,29 @@ type Stream interface {
 
 // watches for all xDS resource types
 type watches struct {
-	endpoints chan cache.Response
-	clusters  chan cache.Response
-	routes    chan cache.Response
-	listeners chan cache.Response
-	secrets   chan cache.Response
-	runtimes  chan cache.Response
-
-	endpointCancel func()
-	clusterCancel  func()
-	routeCancel    func()
-	listenerCancel func()
-	secretCancel   func()
-	runtimeCancel  func()
-
-	endpointNonce string
-	clusterNonce  string
-	routeNonce    string
-	listenerNonce string
-	secretNonce   string
-	runtimeNonce  string
+	endpoints        chan cache.Response
+	clusters         chan cache.Response
+	routes           chan cache.Response
+	listeners        chan cache.Response
+	secrets          chan cache.Response
+	runtimes         chan cache.Response
+	extensionConfigs chan cache.Response
+
+	endpointCancel        func()
+	clusterCancel         func()
+	routeCancel           func()
+	listenerCancel        func()
+	secretCancel          func()
+	runtimeCancel         func()
+	extensionConfigCancel func()
+
+	endpointNonce        string
+	clusterNonce         string
+	routeNonce           string
+	listenerNonce        string
+	secretNonce          string
+	runtimeNonce         string
+	extensionConfigNonce string
```

Comment:
If you remove changes in this file everything will still work out of the box, isn't it?

Reply:
Yes, the ECDS will still work, but we can't use it with the ADS server.

Reply:
I mean pkg/server/v3/server.go, not quite sure about sotw server.

Reply:
yes, I am talking about this specific file :) maybe these can go via generic `responses`: https://github.com/envoyproxy/go-control-plane/pull/400/files#diff-0173d7909b484a388922cc5438a5b94b78b72fe780a531a0ed246bfa940a69f8R101

All other files are actually needed and the problem is - with current build setup it cannot be merged

Reply:
Not sure lacking extension config watcher has other potential effects or not.

Yes, other files are needed, can I change the current build setup and stop copy those files from v2 code?
I think that should be approved by someone.

Reply:
I think nowadays it's up to @kyessenov maybe?